.terraform-docs.yml

```diff
@@ -0,0 +1,4 @@
+# https://terraform-docs.io/user-guide/configuration/
+
+settings:
+  hide-empty: true
```

Comment:
should always include at least this setting it removes the empty 'modules' section which will never show anything